gdk4/Cargo.toml

```diff
@@ -35,6 +36,7 @@ gio = {git = "https://github.com/gtk-rs/gtk-rs-core", version = "0.18", features
 glib = {git = "https://github.com/gtk-rs/gtk-rs-core", version = "0.18", features = ["v2_66"]}
 libc = "0.2"
 pango = {git = "https://github.com/gtk-rs/gtk-rs-core", version = "0.18", features = ["v1_46"]}
+gl = {version = "0.14", optional = true}
```

Comment:
Can you enable this feature in https://github.com/gtk-rs/gtk4-rs/blob/master/.github/workflows/CI.yml#L46 and add `
            --extern-html-root-url=gl=https://docs.rs/gl/latest/` at https://github.com/gtk-rs/gtk4-rs/blob/master/.github/workflows/docs.yml#L69?